sql_metadb/derived_tables/instance_statistical_code.sql

```diff
@@ -0,0 +1,49 @@
+--This derived table extracts data for the instance statistical codes. 
+--It includes the instance uuid, hrid, the statistical code uuid, name, associated code, type id and name.
+--Ordinality has been included.
+DROP TABLE IF EXISTS instance_statistical_codes;
+
+CREATE TABLE instance_statistical_codes AS 
+WITH stcodes AS (
+	SELECT 
+		i.id::uuid AS instance_id,
+		jsonb_extract_path_text(i.jsonb, 'hrid') AS instance_hrid,
+		statcodes.jsonb #>> '{}' AS statistical_code_id,
+		statcodes.ordinality AS stat_code_ordinality
+	FROM 
+		folio_inventory.instance AS i 
+		CROSS JOIN LATERAL jsonb_array_elements(jsonb_extract_path(i.jsonb, 'statisticalCodeIds')) WITH ORDINALITY AS statcodes (jsonb)
+)
+SELECT
+	stc.instance_id,
+	stc.instance_hrid,
+	stc.statistical_code_id,
+	stc.stat_code_ordinality,
+	sct.statistical_code_type_id,
+	sctt.name AS statistical_code_type_name,
+	sct.code AS statistical_code,
+	sct.name AS statistical_code_name,
+	stc.stat_code_ordinality 

```

Comment:
This is a duplication of line 21

Reply:
This isn't a duplication. There's the name for the code type and then also the code.